Cargo.toml

```diff
@@ -7,7 +7,12 @@ authors = ["Emil Sjölander <emil@visly.app>"]
 
 [dev-dependencies]
 criterion = "0.2"
+yoga = "0.3"
 
 [[bench]]
 name = "generated"
-harness = false
\ No newline at end of file
+harness = false
```

Comment:
We already have a `[[bench]]` section. Is it legal toml to have 2?

Reply:
Well, it works, so I guess so.